Sources/_CShims/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+##===----------------------------------------------------------------------===##
+##
+## This source file is part of the SwiftFoundation open source project
+##
+## Copyright (c) 2024 Apple Inc. and the SwiftFoundation project authors
+## Licensed under Apache License v2.0
+##
+## See LICENSE.txt for license information
+## See CONTRIBUTORS.md for the list of SwiftFoundation project authors
+##
+## SPDX-License-Identifier: Apache-2.0
+##
+##===----------------------------------------------------------------------===##
+
+file(GLOB_RECURSE _CShimsSources "*.c")
+add_library(_CShims STATIC ${_CShimsSources})
+
+target_include_directories(_CShims PUBLIC include)
+
+if(CMAKE_SYSTEM_NAME STREQUAL Windows)
+    target_compile_options(_CShims PUBLIC
+        "$<$<COMPILE_LANGUAGE:C,CXX>:SHELL:-D_CRT_SECURE_NO_WARNINGS>")
+endif()
```

Comment:
Do we actually need this? I think that most of the code in Foundation is clean wrt CRT security.